src/nav/mlops.yml

```diff
@@ -4,7 +4,11 @@ pages:
   - title: Introduction to model performance monitoring
     path: /docs/mlops/get-started/intro-mlops
   - title: Bring your own data
-    path: /docs/mlops/bring-your-own/mlops-byo
+    pages:
+      - title: Overview
```

Comment:
Why do we need this "overview" section?
The mlops-byo content can be attached to the "title: Bring your own data"

Reply:
If you want an overarching tab that says, "Bring your own data" then the mlops-byo content will need to have a different title. I am open to other names!

Reply:
This was resolved in a google doc